network/src/peers.rs

```diff
@@ -427,9 +427,9 @@ impl<N: Network, E: Environment> Peers<N, E> {
                 }
 
                 // Skip if the number of connected peers is above the minimum threshold.
-                match number_of_connected_peers < E::MINIMUM_NUMBER_OF_PEERS {
+                match number_of_connected_peers > 0 && number_of_connected_peers < E::MINIMUM_NUMBER_OF_PEERS {
```

Comment:
Is there a reason to only request more peers if we already are connected to peers?

Reply:
I we aren't connected to anyone, we won't be able to send any peer requests; this is just a log cleanup, as we currently report that we are `Sending requests for more peers connections` even if no such request is actually sent (since `self.connected_peers()` returns an empty collection).

Reply:
ah, you meant the early return; I adjusted the logic so that the change only applies to the `true` branch :+1: